homeassistant/components/wiffi/sensor.py

```diff
@@ -38,6 +43,13 @@
     WIFFI_UOM_MILLI_BAR: PRESSURE_MBAR,
 }
 
+# map to determine HA entity category from wiffi's entity name
+NAME_TO_ENTITY_CAT = {
```

Comment:
We should migrate this to use `SensorEntityDescription`s instead.

Reply:
Interesting. Is this a new concept? I didn't find it in the developers docu. Will try.